js/app.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable comma-dangle */
+/* eslint-disable function-paren-newline */
+/* eslint-disable implicit-arrow-linebreak */
```

Comment:
- [ ] **[REQUIRED CHANGES]**

- Please, let's try to write clean codes which conform to standards to make us better developers. try removing these lines and fix the linter error as this is not really a good reason to disable the eslint linter.